### PR TITLE
problem-set-2

### PR DESCRIPTION
**Pull request by ismailsola**

### CODE REVIEW

File: problemset2.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+bedtools flank -l 1000 -r 0 -s -i \
+$datasets/bedtools/tss.hg19.chr22.bed -g $datasets/genome/hg19.genome \
+ | bedtools sort -i - \
+ | bedtools map -a - -b $datasets/bedtools/ctcf.hela.chr22.bg -c 4 -o median \
+ | wc -l 
+
+<<A
```
Comment (jayhesselberth) [inline problemset2.sh]: Something got screwed up in this file. Delete the `<<A` and `<<` lines and recommit.

Comment (ismailsola) [inline problemset2.sh]: I changed the file. Recommitted. Not sure if I need to do anything in github (redo the pull request?). Thx Ismail


________________________________

**Comment by ismailsola** [comment]
OK Jay, Looks like I had an outdated TSS file since PRAME was not in the file. Since I was not sure how to resynch just one file, I re-cloned the whole directory. Good news is that I did find PRAME in the new TSS file. Bad news is that since I re-cloned and my methods of file organization were messy, I have to go back in the .sh file to recreate all the dependant files. Teaches me multiple lessons - learn how to resynch one file, organize so that I have files I create separate from the cloned masters!! Sorry - big learning curve for me.....

**Comment by ismailsola** [comment]
I did get the right answer PRAME after using the right TSS file. I did a new commit with the answers and code file. Not sure if you can see it.

**Comment by jayhesselberth** [comment]
Looks like your `answers.yml` file is actually not YAML, but your run script. Somehow I think your filenames went pear shaped. Anyway I'll take your work for it that you got PRAME. Stick with it.

**Pull request closed without merging**